src/binaries/query/oss_main.rs

```diff
@@ -47,5 +49,7 @@ fn main() {
 async fn main_entrypoint() -> Result<()> {
     let conf: InnerConfig = InnerConfig::load()?;
     init_services(&conf).await?;
+    // init oss license manager
+    OssLicenseManager::init()?;
```

Comment:
this line can be moved into the function init_services?

Reply:
`init_services` will be called in both `oss` and `ee` main, it is used to init all `ee` and `oss` version services, see `ee_main.rs`.

Reply:
it's ok to be called in different places, the `ee` code can choose to override multiple singleton managers on their place.

Reply:
global instance cannot be override, otherwise, it will `panicked at 'value has been set in global data registry'`